mlflow/java/client/src/main/java/org/mlflow/api/proto/ModelRegistry.java

```diff
@@ -10965,6 +10965,32 @@ public GetRegisteredModel parsePartialFrom(
      * <code>optional int64 max_results = 1 [default = 50000];</code>
      */
     long getMaxResults();
+
+    /**
+     * <pre>
+     * Pagination token to go to the next page based on a previous query.
+     * </pre>
+     *
+     * <code>optional string page_token = 2;</code>
+     */
+    boolean hasPageToken();
```

Comment:
Nit: `page` is a word and not really understood as an abbreviation for `pagination`. Perhaps `hasPaginationToken` would be better?

Reply:
This might be a larger discussion. Feel free to just say no here because it's already been implemented this way in other parts of our code.

Reply:
It's definitely worth considering!

For the time being, though, we do have `page_token` pretty much everywhere in our paginated API. It does appear to be somewhat? canonical in other APIs, like [this](https://googleapis.dev/python/google-api-core/latest/page_iterator.html) and [this](https://developers.google.com/google-ads/api/docs/concepts/retrieving-objects)

Reply:
Oh interesting! I think it's fine then if Google is doing this convention.